calico/getting-started/kubernetes/vpp/openshift.mdx

```diff
@@ -0,0 +1,211 @@
+---
+description: Install Calico VPP on an OpenShift 4 cluster.
+---
+
+# Install an OpenShift 4 cluster with Calico VPP
+
+## Big picture
+
+Install an OpenShift 4 cluster with {{prodname}} VPP.
+
+## Value
+
+Augments the applicable steps in the [OpenShift documentation](https://cloud.redhat.com/openshift/install)
+to install {{prodname}}.
+
+## How to
+
+### Before you begin
+
+- Ensure that your environment meets the {{prodname}} [system requirements](../openshift/requirements.mdx).
+
+- **If installing on AWS**, ensure that you have [configured an AWS account](https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-account.html) appropriate for OpenShift 4,
```

Comment:
Here we have an optional requirement (for AWS). But the other requirements all point to OCP AWS install docs. 
Can we point instead to generic procedures?

Reply:
I based my doc on the **Calico openshift** doc (https://docs.tigera.io/calico/latest/getting-started/kubernetes/openshift/installation) and I think **Calico openshift** at the moment only supports installing on **AWS**. I see that the **Calico openshift** doc has been cleaned up a bit since the last time I checked :)

Anyways, I have changed the wording in my doc to comply with the latest **Calico openshift** doc's and I am explicitly stating that we only support installs on AWS at the moment.